src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/V2FeedListResourceProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Protocol.Core.Types;
+using NuGet.Protocol.LocalRepositories;
+
+namespace NuGet.Protocol.LegacyFeed
```

Comment:
>LegacyFeed [](start = 25, length = 10)

Remove this last part of the namespace. Everything is under NuGet.Protocol if i recall